pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -55,12 +60,16 @@ public JwtAuthenticator(final String secret) {
     public void validate(TokenCredentials credentials) {
         CommonHelper.assertNotBlank("secret", secret);
 
-        final String token = credentials.getToken();
+        String token = credentials.getToken();
         boolean verified = false;
         SignedJWT signedJWT = null;
 
         try {
             // Parse the token
+            if (Base64.isBase64(token)) {
```

Comment:
An interesting idea, but I'm not very enthusiastic about magic "behaviour": don't you think we could add new methods for generic JWT tokens: `validateToken(final String token)` and `validateBase64Token(final String bas64Token)`?